iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/CalculatedBusImpl.java

```diff
@@ -26,9 +24,44 @@
 
     private final List<NodeTerminal> terminals;
 
-    CalculatedBusImpl(String id, VoltageLevelExt voltageLevel, List<NodeTerminal> terminals) {
+    private NodeTerminal terminalRef;
+
+    CalculatedBusImpl(String id, NodeBreakerVoltageLevel voltageLevel, TIntArrayList nodes, List<NodeTerminal> terminals) {
         super(id, voltageLevel);
         this.terminals = Objects.requireNonNull(terminals);
+        this.terminalRef = findTerminal(voltageLevel, nodes, terminals);
+    }
+
+    /**
+     * Find a terminal to retrieve voltage, angle or connected and synchronous component numbers.
+     * If the terminals list contains at least one element, we use the first terminal as reference.
+     * Otherwise this method try to find a terminal which does not belong to this bus, but to the same "electrical" bus
+     * and therefore can be used as reference.
+     *
+     * @param voltageLevel The {@literal VoltageLevel} instance to traverse
+     * @param nodes The nodes belong to this bus
+     * @param terminals The terminals belong to this bus
+     * @return The first terminal of the {@param terminals} list, or a terminal which belongs to an equivalent "electrical" bus.
```

Comment:
I think the inline {@param ***} does not work in javadoc, unfortunately.

Reply:
Replaced by {@code }